jobs-builder/jobs/dependencies.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file contains the configurations to generate the Kernel and QEMU jobs.
+#
+---
+###
+# Define shareable YAML snippets.
+###
+- common_job_properties: &common_job_properties
+    name: 'common_job_properties'
+    project-type: freestyle
+    disabled: false
+    concurrent: true
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 5
+    # Convert the os variable to label name.
+    node: !include-jinja2: include/os2node.yaml.inc
+    scm:
+      - git:
+          url: https://github.com/kata-containers/kata-containers
+          branches:
+            - '*/{branch}'
+    wrappers:
+      - ansicolor:
+          colormap: "xterm"
+      - openstack:
+          single-use: True
+      - timestamps
+      - timeout:
+          timeout: 20
+          type: no-activity
+- default_publishers: &default_publishers
+    name: 'default_publishers'
+    publishers:
+      - archive:
+          artifacts: "artifacts/*"
+###
+# Define jobs templates.
+###
+- job-template:
+    name: kata-containers-{branch}-kernel-{flavor}-{arch}
+    <<: *common_job_properties
+    maintainers:
+      - Wainer dos Santos Moschetta &ltwainersm@redhat.com&gt
```

Comment:
I'm still not sure about this. I appreciate this is your work, but shouldn't this really be a more generic contact for the Kata project (or just remove `maintainers:`?)

Reply:
I will change to a more generic contact :)

Reply:
done